Source/Orts.Simulation/Common/Scripting/TrainControlSystem.cs

```diff
@@ -692,16 +701,20 @@ public enum MonitoringStatus
     public struct SignalFeatures
     {
         public readonly string MainHeadSignalTypeName;
+        public readonly string SignalTypeName;
```

Comment:
No, I don't think so because these struct objects are used to pass information to the script. Adding fields does not cause a problem but removing or changing fields does.

The other way this can break compatibility is if the scripts creates its own SignalFeatures object. Carlo didn't seem to say that this was breaking something in his scripts after my response earlier so another constructor is not needed.